### PR TITLE
Add latest link to site

### PR DESCRIPTION
**Pull request by dg-ratiodata**
@myronmarston Now you can have `/documentation/latest/rspec-expectations` (http://localhost:4567/documentation/latest/rspec-expectations/). Referencing rspec/rspec-expectations#823 here. I used `middleman`'s proxy-method for this. This requires a valid entry in the sitemap as destination. This only works for `index.html`. Ok?

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -96,133 +98,9 @@ def deploy_to target
...
-    'u2028' => "\u2028",
-    'u2029' => "\u2029"
-  }
-  JS_ESCAPES = JS_UNESCAPES.invert
-  JS_ESCAPE_PATTERN = Regexp.union(JS_ESCAPES.keys)
-  def js_escape(string)
-    string.gsub(JS_ESCAPE_PATTERN) { |match| '\\' + JS_ESCAPES[match] }
-  end
+RSpecInfo::Helpers.rspec_documentation_latest(source_dir).each do |gem_name, version|
+  proxy "/documentation/latest/#{gem_name}/", "/documentation/#{version}/#{gem_name}/index.html"
```
Comment (myronmarston) [inline config.rb]: Playing around with this locally, it looks like this works for the root README page for each gem but not for any pages under that. For rspec/rspec-expectations#823, we really just want http://rspec.info/documentation/latest/rspec-expectations/RSpec/Matchers/MatcherProtocol.html to proxy or redirect to http://rspec.info/documentation/3.3/rspec-expectations/RSpec/Matchers/MatcherProtocol.html but that's not working right now.

Can you think of a way to make this work for all the doc pages?

Also, redirects would be better than serving the same content at a second URL.  Of course, with a static site generator you can't choose HTTP status codes to respond with to do a proper redirect but we could use a meta refresh tag.  Thoughts?

/cc @JonRowe

**Comment by dg-ratiodata** [comment]
What about this one?

**Comment by dg-ratiodata** [comment]
> Can you think of a way to make this work for all the doc pages?

I tried a few ones and it worked for me.

> Also, redirects would be better than serving the same content at a second URL. 

Fixed.

**Comment by JonRowe** [comment]
Do you think your could split the commits for the refactoring vs the additional redirect? bit hard to understand in one chunk

**Comment by dg-ratiodata** [comment]
Better?

**Comment by dg-ratiodata** [comment]
This was not a perfect split. this is new: https://github.com/dg-ratiodata/rspec.github.io/commit/599dab46be37940d58e8c4d18aba88f68a452369#diff-6050de8c34f91c732c56fe76c1374b7cR24

File: source/redirect-latest.html.slim [hunk 2]
```diff
@@ -0,0 +1,5 @@
+- content_for(:senarios) do
```
Comment (myronmarston) [inline source/redirect-latest.html.slim]: s/senarios/scenarios/

...although, why call it that?

Comment (maxmeyer) [inline source/redirect-latest.html.slim]: Oh. C&P error. Sorry

Comment (myronmarston) [inline source/redirect-latest.html.slim]: No worries :).  Mind fixing it?  Seems necessary for it to work properly.  I'll be happy to merge once that's fixed.

Comment (maxmeyer) [inline source/redirect-latest.html.slim]: Jup... In the next 30 mins I think... I just need to finish my breakfast ;-).

Comment (myronmarston) [inline source/redirect-latest.html.slim]: You must be in a different time zone.  It's 11 PM here and I'll be going to bed shortly :).

Comment (JonRowe) [inline source/redirect-latest.html.slim]: Well it's almost 5pm on friday here ;)

Comment (dg-ratiodata) [inline source/redirect-latest.html.slim]: Ok. Fixed.

Comment (dg-ratiodata) [inline source/redirect-latest.html.slim]: Sure, Germany: http://www.timeanddate.com/worldclock/germany/berlin :smile:

**Comment by dg-ratiodata** [comment]
Thanks.

**Comment by myronmarston** [comment]
It's been deployed:

http://rspec.info/documentation/latest/rspec-expectations/RSpec/Matchers/MatcherProtocol.html

**Comment by dg-ratiodata** [comment]
Great.

**Pull request merged**